reactor-client/pom.xml

```diff
@@ -0,0 +1,195 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.aerospike</groupId>
+  <artifactId>aerospike-reactive-client</artifactId>
+  <version>4.1.8</version>
+  <packaging>jar</packaging>
+
+  <name>aerospike-reactor-client</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <reactor.version>3.1.8.RELEASE</reactor.version>
+    <assertj.version>3.9.0</assertj.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.aerospike</groupId>
+      <artifactId>aerospike-client</artifactId>
+      <version>4.1.8</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport</artifactId>
+      <version>4.1.11.Final</version>
```

Comment:
please extract versions to placeholders.
Also could you please clarify why reactor dependencies have compile scope whereas netty provided?

Reply:
fixed